source/trackers.json

```diff
@@ -1072,6 +1072,7 @@
         "adjust.net.in": "adjust",
         "adjust.world": "adjust",
         "apptrace.com": "adjust",
+        "adobe.net": "adobe_experience_cloud",
```

Comment:
Doesn't this domain belong to an audience manager more ?
```
"adobe_audience_manager": {
            "name": "Adobe Audience Manager",
            "categoryId": 4,
            "url": "http://www.demdex.com/",
            "companyId": "adobe"
        },
```
It seems that this tracker also has the wrong website 
https://business.adobe.com/products/audience-manager/adobe-audience-manager.html